app/controllers/themes_controller.rb

```diff
@@ -75,4 +80,18 @@ def theme_params
   def ensure_admin!
     authorize! :read, :admin_dashboard
   end
+
+  def apply_theme_to_site
+    theme = Theme.current_theme
+    appearance_to_theme_mapping = {
+      header_background_color:          theme.primary_color,
+      header_text_color:                theme.primary_text_color,
+      link_color:                       theme.accent_text_color,
+      footer_link_color:                theme.accent_text_color,
+      primary_button_background_color:  theme.primary_color
+    }
```

Comment:
@norm0 - The mappings above this line probably need refinement, but I wanted to get this out so we had the basic framework in place.